classes/class.memberorder.php

```diff
@@ -21,6 +18,25 @@ function __construct($id = NULL)
 				return $this->getEmptyMemberOrder();	//blank constructor
 		}
 
+		public function __get( $name )
+		{
+			if( 'Gateway' === $name ){
+				// we are here only if ->Gateway is NOT set
+				$this->setGateway( $this->gateway ? $this->gateway : '' );
```

Comment:
Use empty string if $this->gateway is null, to be sure the `->Gateway` instance will be set (to the test gateway) and avoid fatals.